src/components/elements/accordion.tsx

```diff
@@ -58,8 +62,16 @@ const AccordionWrapper = styled.div`
 `
 const TRANSITION_DURATION = 250
 
+type AccordionProps = {
+    children?: ChildType | ChildType[]
+    has_single_state?: boolean
+    id?: string
+    is_default_open?: boolean
+    nodes?: ReactNode[]
+}
+
```

Comment:
Please check here and further in PR if we really need props to be all as optional.
will components work without all these props, marked as unnecessary?

Reply:
Was decided to remove whole component, because we use him...nowhere